### DISCUSSION TITLE
High quality covers

### DISCUSSION THREAD

**Discussion by niranjanvram** [cinemagoer/cinemagoer#512] [Q&A]
I had been using imdbpy for a descent time, but due to its change into cinemagoer, I recently switched to it. I am using cinemagoer for fetching imdb information of any movie/series and providing this info to users on telegram through a bot. I am having a problem of fetching cover arts/posters with low quality. I would like to have high quality posters to be returned to the variable. This is how I specify it in my code:

release_date=imdb['release_date'], year=imdb['year'], genres=imdb['genres'], poster=imdb['cover url']

Is it cover url or poster or anything else? The following is the output for the poster variable if the query was "premalu"(the poster quality is low as seen in below image):

![premalu imdb](https://github.com/cinemagoer/cinemagoer/assets/83854040/2482ad34-49a7-486d-abb5-9abd4532accd)

Am I doing something wrong? Please help me resolve the same. Thank you.

**Comment by alberanid** [comment]
This discussion is no longer relevant due to issue https://github.com/cinemagoer/cinemagoer/issues/557: as of June 2026 Cinemagoer no longer analyzes web pages and can only rely on non-commercial datasets.
